redaxo/src/addons/backup/lib/backup.php

```diff
@@ -329,6 +329,8 @@ public static function exportDb($filename, array $tables = null)
             }
         }
         foreach ($tables as $table) {
+            $table = preg_replace('/[^a-z0-9._-]/i', '', $table);
```

Comment:
Hier müsste man ggf noch ein paar zeichen erlauben, falls jmd andere als die genannten in tabellenbezeichnern verwendet